src/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -83,6 +83,18 @@ public TimeSpan UserProcessorTime
             }
         }
 
+        partial void EnsureHandleCountPopulated()
+        {
+            if (_processInfo.HandleCount <= 0 && _haveProcessId)
+            {
+                string path = "/proc/" + _processId.ToString() + "/fd";
```

Comment:
For consistency with other part of the Linux code that uses procfs, consider adding a function to https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs#L14 that produces this path.